methods.go

```diff
@@ -205,12 +207,72 @@ func decodeTorrentsStatusResponse(resp *DelugeResponse) (map[string]*TorrentStat
 	return result, nil
 }
 
+// TorrentError is a tuple of a torrent and an error message, returned by
+// methods that manipulate many torrents at once.
+type TorrentError struct {
+	// ID is the hash of the torrent that experienced an error
+	ID string
+	// Message is a string message
+	Message string
+}
+
+// Error satisfies the error interface
+func (t TorrentError) Error() string {
+	return fmt.Sprintf("<%s>: '%s'")
```

Comment:
Something is missing here.

Reply:
Nice catch!

Reply:
You can actually have the Go linter show these in your IDE of choice.